src/z3c/pagelet/zcml.py

```diff
@@ -92,13 +92,6 @@ def pageletDirective(
     # Get the permission; mainly to correctly handle CheckerPublic.
     permission = viewmeta._handle_permission(_context, permission)
 
-    # The class must be specified.
-    if not class_:
-        raise ConfigurationError("Must specify a class.")
-
-    if not zope.interface.interfaces.IInterface.providedBy(provides):
-        raise ConfigurationError("Provides interface provide IInterface.")
-
```

Comment:
This doesn't seem to be related to Python version support changes?

Reply:
Did you remove dead code after adding those tests and seeing those errors get caught earlier?

Reply:
(Ah, yes, I should've looked at the actual commit messages.  I blame GitHub for making commit-by-commit reviews more difficult than all-at-once reviews.)